components/eam/docs/user-guide/namelist_parameters.md

```diff
@@ -156,3 +156,20 @@
 | Parameter                 | Description                                                       | Default value          |
 | ------------------------- | ----------------------------------------------------------------- | ---------------------- |
 | `cosp_lite`       | This namelist sets cosp_ncolumns=10 and cosp_nradsteps=3 (appropriate for COSP statistics derived from seasonal averages), and runs MISR, ISCCP, MODIS, and CALIPSO lidar simulators (cosp_lmisr_sim=.true.,cosp_lisccp_sim=.true., cosp_lmodis_sim=.true.,cosp_llidar_sim=.true.).  | `false`                |
+
+## Orographic drag schemes
+
+| Parameter                 | Description                                                       | Default value          |
+| ------------------------- | ----------------------------------------------------------------- | ---------------------- |
+| `use_gw_oro`       | This namelist controls the default linear orographic gravity wave drag (oGWD) for E3SM, if used, the default oGWD is turned on.                                | `true`                |
+| `do_tms`           | This namelist controls the default Turbulent Mountain Stress (TMS) for E3SM, if used, the default TMS is turned on.                                 | `false`                |
+| `effgw_oro`        | Efficiency associated with orographic gravity waves.                                                                     | `0.375`                |
+| `tms_orocnst`      | Turbulent mountain stress parameter used when TMS calculation is turned on             | `1.0`                |
+| `tms_z0fac`        | Factor determining z_0 from orographic standard deviation [ no unit ] for TMS.                                                   | `0.75`                |
+| `use_od_ls`        | This namelist controls the new nonlinear oGWD, if used, the nonlinear oGWD is turned on. use_od_ls should not be used at the same time with use_gw_oro.                  | `true`                |
```

Comment:
Default value should be `false` since it's going in as a stealth feature. We will update this to `true` if and when we turn it on by default. Same goes with `use_od_bl`, `use_od_ss`, and `use_od_fd`